pkg/driver/sanity_test.go

```diff
@@ -294,8 +294,8 @@ func (c *fakeCloudProvider) ListSnapshots(ctx context.Context, volumeID string,
 func (c *fakeCloudProvider) ResizeDisk(ctx context.Context, volumeID string, newSize int64) (int64, error) {
 	for volName, f := range c.disks {
 		if f.Disk.VolumeID == volumeID {
-			c.disks[volName].CapacityGiB = newSize
-			return newSize, nil
+			c.disks[volName].CapacityGiB = util.BytesToGiB(newSize)
```

Comment:
note that `BytesToGiB` will round down in GiB.
Maybe ByteToGib should return a float value and let the caller choose to either round up or down.

In most CSI case, I would suspect we want to round up as we want the data to fit in the disk.

Reply:
Indeed, will use `RoundUpGib` instead, thank you